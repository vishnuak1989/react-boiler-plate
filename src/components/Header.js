import React from 'react'
import { NavLink } from 'react-router-dom'

const Header=()=>(
  <header className="header">
  <div className="header__content">
    <h1 className="header__title">TITLE</h1>
    <ul>
    <NavLink to="/" activeClassName="selected" exact={true}><button className="button">Dashboard</button></NavLink>
    <NavLink to="/about" exact={true}><button className="button">About</button></NavLink>
    </ul>
    </div>
  </header>
)

export default Header;
