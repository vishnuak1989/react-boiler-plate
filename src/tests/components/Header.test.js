import React from 'react'
// import Reactshallowrenderer from 'react-test-renderer/shallow';
import {shallow} from 'enzyme';
import Header from '../../components/Header'

test("should render correctly",()=>{
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();

})
