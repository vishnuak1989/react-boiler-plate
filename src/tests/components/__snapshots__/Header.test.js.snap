// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<header
  className="header"
>
  <div
    className="header__content"
  >
    <h1
      className="header__title"
    >
      TITLE
    </h1>
    <ul>
      <NavLink
        activeClassName="selected"
        ariaCurrent="true"
        exact={true}
        to="/"
      >
        <button
          className="button"
        >
          Dashboard
        </button>
      </NavLink>
      <NavLink
        activeClassName="active"
        ariaCurrent="true"
        exact={true}
        to="/about"
      >
        <button
          className="button"
        >
          About
        </button>
      </NavLink>
    </ul>
  </div>
</header>
`;
